data/human/culture conversations.txt

```diff
@@ -615,3 +615,68 @@ mission "Sol-Bound Shuttle"
 			label end
 			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
 				decline
+
+mission "Burthen Triathlon"
+	minor
+	source
+		planet "Burthen"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Burthen's planetary spaceport is not known to be well-trafficked because of the planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are now occupied by shuttlecraft, and as you step out onto the spaceport's rather untarnished apron, you notice there's a sizable crowd at the edge of the spaceport, some of them obviously from offworld judging by their height. There's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`
```

Comment:
```suggestion
			`Burthen's planetary spaceport is not a common destination thanks to the planet's high gravity, but today it seems to have a lot of visitors. Many of the usually empty landing pads are occupied by shuttlecraft and you notice there's a sizable crowd at the edge of the spaceport as you step out onto its rather untarnished apron. Some members of the crowd are obviously from offworld judging by their height, and there's even a few media crews standing by, although it looks like a couple of their camera drones are struggling in Burthen's gravity.`
```

Well-trafficked implies drugs or similar. The second sentence is also a bit of a grammatical mess.